cli/admin/src/main/java/io/pravega/cli/admin/segmentstore/tableSegment/ListTableSegmentKeysCommand.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.cli.admin.segmentstore.tableSegment;
+
+import io.pravega.cli.admin.CommandArgs;
+import io.pravega.cli.admin.utils.AdminSegmentHelper;
+import io.pravega.client.tables.impl.HashTableIteratorItem;
+import io.pravega.client.tables.impl.TableSegmentKey;
+import io.pravega.shared.protocol.netty.PravegaNodeUri;
+import lombok.Cleanup;
+import org.apache.curator.framework.CuratorFramework;
+
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.stream.Collectors;
+
+public class ListTableSegmentKeysCommand extends TableSegmentCommand {
```

Comment:
How do we handle the situation when there are tens of thousands of keys?

Reply:
`keyCount` is the upper limit on the total keys that are to be listed. So we would get at most `keyCount` number of keys.